README.md

```diff
@@ -65,52 +65,12 @@ new KinesisWritable({
 
 `streamName` is the name of the Kinesis Stream.
 
-### Methods
-
-* `getStreamName()`: returns the stream's name.
-* `setStreamName(name)`: set the name of the stream where messages will be send.
-
 ### Events
 
-* `errorRecord`: Emitted once for each failed record at the `aws.kinesis.putRecords`'s response.
```

Comment:
Have you checked if we use this event in `auth0-tenant-audit` / `auth0-worker-lib`?